check_if_image_tag_exists/test_config.json

```diff
@@ -1,4 +1,5 @@
 {
+	"schemaVersion": "1.0.0",
 	"commandTests": [
```

Comment:
this was more of a proof of concept to show that multiple versions could be supported (as well as testing). you're definitely right though, we should try and keep it as simple as possible while we can. i'll remove the earlier schema.